### ISSUE TITLE
dft in ask_int()

### ISSUE THREAD

**Issue by arurke** [jeffrimko/Qprompt#2]
Apologies if this is something obvious - still pretty new to python.

When calling
```
test = qprompt.ask_int("test", dft=0)
```
I get
```
File "/usr/local/lib/python3.4/dist-packages/qprompt.py", line 240, in ask
    vld = sorted(list(set([fmt(v) if fmt(v) else v for v in vld])))
TypeError: unorderable types: type() < int()
```
which is caused by the dft value. I tried int(0), "0",  and some other combination with no luck. Bug?

Python 3.4.3. Qpromtp 0.8.1.

**Comment by jeffrimko** [comment]
That is definitely a bug. Seems to be specific to Python3. Thanks for finding it! I will fix and publish a new release shortly.

**Comment by arurke** [comment]
np.

Curious: How long does it take for a new version to appear in pip?

**Comment by jeffrimko** [comment]
Just published the new release, version `0.8.2`. Added you to the new [AUTHORS file](https://github.com/jeffrimko/Qprompt/blob/master/AUTHORS.adoc).

Think it should show up on pip immediately. Try running `pip install --upgrade qprompt` to see if the new release is pulled down.

**Comment by arurke** [comment]
Thanks! Tested and works 👍
